panel/io/server.py

```diff
@@ -455,6 +456,7 @@ async def get(self, *args, **kwargs):
             if authorized is None:
                 return
             elif not authorized:
+                self.set_status(403)
```

Comment:
I don't know if we should set it to 200 outside this if statement.

Reply:
Don't think so, that should be handled by the response.